spyder/utils/debug.py

```diff
@@ -33,7 +33,7 @@ def log_last_error(fname, context=None):
         print("Context", file=fd)
         print("-------", file=fd)
         print("", file=fd)
-        print(context, file=fd)
+        print(u' '.join(context).encode('utf-8').strip(), file=fd)

```

Comment:
Then please apply this change only for Python 2 and leave the previous line for Python 3